src/devices/bus/hp_dio/hp98543.cpp

```diff
@@ -154,10 +154,24 @@ WRITE16_MEMBER(dio16_98543_device::vram_w)
 
 uint32_t dio16_98543_device::screen_update(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect)
 {
+	int startx[TOPCAT_COUNT], starty[TOPCAT_COUNT];
+	int endx[TOPCAT_COUNT], endy[TOPCAT_COUNT];
+
+	for(int i = 0; i < TOPCAT_COUNT; i++)
```

Comment:
Can you please be consistent with spacing?  You've got `for(` and `for (` in the same function now, where you seem to mostly use `if (` most of the time.  You've even got both in this change, and it's not like you're modifying someone else's code.  It looks really careless.